netkan/netkan/metadata.py

```diff
@@ -59,10 +59,16 @@ def hook_only(self):
             return False
         return self.on_spacedock
 
+    def string_attrib(self, val):
+        return {
+            'DataType': 'String',
+            'StringValue': val,
+        }
+
     def sqs_message_attribs(self, ckan_group=None):
         attribs = {}
         if ckan_group and not getattr(self, 'x_netkan_allow_out_of_order', False):
-            attribs['HighestVersion'] = ckan_group.highest_version().string
+            attribs['HighestVersion'] = self.string_attrib(ckan_group.highest_version().string)
         return attribs
```

Comment:
We should open an issue to refactor this so that it does the RightThings&trade; - There is definitely a nicer way to do this.

Reply:
Yeah, this was just a safe short term fix to get it running again, please feel free to change it to whatever would be best.